meson.build

```diff
@@ -0,0 +1,33 @@
+#
+# See COPYING for more information about licensing
+#
+
+project('nethogs',['c','cpp'],version : '0.8.7',
```

Comment:
specifying '0.8.7' here seems a bit weird, since the current main branch is '0.8.7 and a bit'. Is there any convention in the meson ecosystem to deal with the version differently, perhaps similar to what the Makefile does?

Also, the formatting seems somewhat haphazard (e.g., when do use newlines and when not). Is there any formatting tool for meson build files? Did you use it?

Reply:
Unfortunately there is not a tool, but here is the style guide: https://mesonbuild.com/Style-guide.html

Let me see how I can fix the version. Probably I can get it from your version script

Reply:
I have added the detection of the version from the tag as in the Makefiles.